packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -110,8 +110,7 @@ export class StatusBarImpl extends ReactWidget implements StatusBar {
 
     protected internalSetBackgroundColor(color?: string): void {
         this.backgroundColor = color;
-        // eslint-disable-next-line no-null/no-null
-        this.node.style.backgroundColor = this.backgroundColor ? this.backgroundColor : null;
+        this.node.style.backgroundColor = this.backgroundColor || '';
```

Comment:
Default is [`transparent`](https://www.w3schools.com/cssref/pr_background-color.asp)